lib/checks/keyboard/main-is-top-level.js

```diff
@@ -0,0 +1,13 @@
+var landmarks = axe.commons.aria.getRolesByType('landmark');
+var parent = node.parentElement;
```

Comment:
This PR has been open for a while now. Since than, we've merged shadow DOM in. It has a different method of getting at parent nodes. Please use `axe.commons.dom.getComposedParent()`. That way this check will look passed shadow DOM boundaries. You'll have to rebase to get the latest on develop into this branch.

Reply:
@jongund This line still uses `node.parentElement`, please fix this too.